app/views/huawei_router_passwords/new.html.erb

```diff
@@ -1,8 +1,8 @@
 <%- title = t('page_titles.your_huawei_router_password') %>
 <% content_for :title, title %>
 <%- content_for :before_content do %>
-  <% breadcrumbs([{ "Home" => root_path },
-                  { t('page_titles.signed_in_internet_access') => last_breadcrumb_path_for_huawei },
```

Comment:
Can this i18n key (`page_titles.signed_in_internet_access`) be removed now? Of repurposed for the new copy?

Reply:
Great spot, but that i18n key is still used elsewhere

Reply:
Ah good to know! 👍🏽